### ISSUE TITLE
kendo-angular-dropdowns: bundling of es modules failed with SyntaxError

### ISSUE THREAD

**Issue by rstabenow** [telerik/kendo-angular#267]
Bundling the es modules with SystemJS Builder failed with message:

```
Unhandled rejection SyntaxError: ../node_modules/@progress/kendo-angular-dropdowns/dist/es/error-messages.js: 'import' and 'export' may appear only with 'sourceType: module' (8:4)
   6 | /**
   7 |  * @hidden
>  8 |  */ export var MultiselectMessages = {
     |     ^
   9 |     'array': 'Expected initial values of array type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection',
  10 |     'object': 'Expected initial values of Object type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection',
  11 |     'primitive': 'Expected initial values of primitive type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection'
```

It works fine with 0.27.0 and earlier, but not with 0.27.1 - 0.27.5.

**Comment by rusev** [comment]
Thank you for reporting the issue. It is related to the [bug reported here](https://github.com/Microsoft/TypeScript/issues/12979).

We will update the `@progress/kendo-angular-dropdowns` package in order to fix the _**es**_ bundle.

**Comment by rusev** [comment]
You can try the latest package version. `0.27.6` should address this issue.

**Comment by rstabenow** [comment]
Works fine with 0.27.6, thank you!

**Comment by b-hobbs** [comment]
I'm having this same issue with 0.38.0, 0.37.0 works though.

**Comment by rstabenow** [comment]
Yes, it's back, same error as before with duplicate comments:

> Unhandled rejection SyntaxError: ../node_modules/@progress/kendo-angular-dropdowns/dist/es/error-messages.js: 'import' and 'export' may appear only with 'sourceType: module' (8:4)
   6 | /**
   7 |  * @hidden
>  8 |  */ export var MultiselectMessages = {
     |     ^
   9 |     'array': 'Expected initial values of array type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection',
  10 |     'object': 'Expected initial values of Object type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection',
  11 |     'primitive': 'Expected initial values of primitive type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection'


Source:
> /**
 * @hidden
 */
/* tslint:disable:max-line-length */
/* tslint:disable:variable-name */
/**
 * @hidden
 */ export var MultiselectMessages = {
    'array': 'Expected initial values of array type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection',
    'object': 'Expected initial values of Object type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection',
    'primitive': 'Expected initial values of primitive type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection'
};

**Comment by rstabenow** [comment]
Same problem with `@progress/kendo-angular-inputs`

**Comment by tsvetomir** [comment]
What project template are you using? We're having troubles reproducing the error locally.

**Comment by b-hobbs** [comment]
@tsvetomir I'm using the [angular-seed](https://github.com/mgechev/angular-seed) by mgechev for the template.
I already have a [fork](https://github.com/b-hobbs/angular-seed) with just kendo dropdowns if you want to use it.
To replicate with the fork just `npm install && npm run serve.prod.aot`

> [13:42:01] SyntaxError in plugin 'run-sequence(serve.prod.aot)'
Message:
    Error compiling global module "@progress/kendo-angular-dropdowns/dist/es/error-messages.js" at node_modules/@progress/kendo-angular-dropdowns/dist/es/error-messages.js
	node_modules/@progress/kendo-angular-dropdowns/dist/es/error-messages.js: 'import' and 'export' may appear only with 'sourceType: module' (8:4)
Details:
    pos: 110
    loc: [object Object]
    _babel: true
    codeFrame:    6 | /**
   7 |  * @hidden
>  8 |  */ export var MultiselectMessages = {
     |     ^
   9 |     'array': 'Expected initial values of array type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection',
  10 |     'object': 'Expected initial values of Object type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection',
  11 |     'primitive': 'Expected initial values of primitive type. See http://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/#value-selection'

**Comment by tsvetomir** [comment]
The issue is resolved with the latest package version (0.38.3). Thanks!
